### PR TITLE
add docs question to PR template

### PR DESCRIPTION
**Pull request by briandowns**
#### Proposed Changes ####

Adds an HTML comment to the Pull Request template asking if the change submitted requires an update to the documentation.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 <!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. -->
 
+<!-- Does this change require an update to documentation? -->
```
Comment (brandond) [inline .github/PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
- [ ] This change requires an update to documentation.
```

**Review by brandond — APPROVED** [review]
Maybe make it a full-on checkbox rather than just a suggested topic?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (briandowns) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: This was agreed to be a comment. I'm happy to change it but can you sign off @davidnuzik

Comment (davidnuzik) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: I vote we just make it a comment. I like the idea of the checkbox -- but to be honest no one seems to use them. I use them in my gh issues for releases and no one checks them off as we go. I say go with the HTML comment for now. If it doesn't seem to be helping we can revisit this and modify as needed later.

**Review by davidnuzik — APPROVED** [review]
LGTM

**Pull request merged**